catalog/imaging/catalog-imaging-transformer/src/main/java/com/connexta/alliance/transformer/nitf/NitfInputTransformer.java

```diff
@@ -59,7 +59,7 @@
 
     private static final String ID = "ddf/catalog/transformer/nitf";
 
-    private static final String MIME_TYPE = "image/nitf";
+    static final String MIME_TYPE = "image/nitf";
```

Comment:
The MimeType ctor throws an exception, so it would have to go into a static initializer block, but would the block do if it couldn't create the MimeType instance? Throw a runtime exception?